core/cloudflow-akka-tests/src/test/java/cloudflow/akkastream/javadsl/AkkaStreamletTest.java

```diff
@@ -117,6 +157,48 @@ public void run() {
     }
   }
 
+  class FileWritingProcessor extends AkkaStreamlet {
+
+    private final String volumeMountName;
+
+    FileWritingProcessor(String volumeMountName) {
+      this.volumeMountName = volumeMountName;
+    }
+
+    AvroInlet<Data> inlet = AvroInlet.<Data>create("in", Data.class);
+    AvroOutlet<Data> outlet = AvroOutlet.<Data>create("out", Data.class);
+
+    public StreamletShape shape() {
+      return StreamletShape.createWithInlets(inlet).withOutlets(outlet);
+    }
+
+    @Override
+    public VolumeMount[] defineVolumeMounts() {
+      return new VolumeMount[] {VolumeMount.createReadWriteMany(volumeMountName, "path")};
+    }
+
+    public AkkaStreamletLogic createLogic() {
+      Path mountedPath = getContext().getMountedPath(defineVolumeMounts()[0]);
+      return new AkkaStreamletLogic(getContext()) {
+        public void run() {
+          getSourceWithOffsetContext(inlet)
+              .via(
+                  Flow.<Pair<Data, CommittableOffset>>create()
+                      .map(
+                          dataIn -> {
+                            Files.write(
+                                mountedPath,
+                                dataIn.first().toString().getBytes(),
+                                StandardOpenOption.SYNC);
+                            return dataIn;
+                          }))
+              .to(getSinkWithOffsetContext(outlet))
```

Comment:
FYI: that method is deprecated. Use `committableSink(outlet)` instead.

Reply:
Done.